src/main/webapp/scripts/common.js

```diff
@@ -24,17 +24,23 @@ window.onscroll = function() {
 
 window.onload = function() {
   fetch("/login").then(result => result.json()).then((details) => {
+    const loginPageURL = window.location.protocol + "//" + window.location.host + "/";
+    console.log(window.location.href);
```

Comment:
This was only for debugging purposes I assume, shall we remove it? Same below :)